client/my-sites/stats/site.jsx

```diff
@@ -380,6 +380,7 @@ class StatsSite extends Component {
 					<div className={ moduleListClasses }>
 						<StatsModule
 							path="posts"
+							gridArea="posts"
```

Comment:
Can we drop this in favor of path?

Reply:
That's for making the grid layout to work properly, which sometimes is different from path